pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -47,6 +47,16 @@ social:
     margin: 0px;
     padding: 0px;
   }
+  #fullwidth0, #fullwidth1, #fullwidth2, #fullwidth3, #fullwidth4 {
```

Comment:
Could also use a CSS class here.

Reply:
Good call, fixed in 62747a5e377538ca9837e47cfab1203de3c03579.